src/components/pages/computing/NewInstancePage/cmp.tsx

```diff
@@ -564,14 +366,37 @@ export default function NewInstancePage({ mainRef }: PageProps) {
                 {!node && (
```

Comment:
Disable button in case we are on hold tier, instead switching automatically to pay